demo/list-item.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export default class ListItem extends React.Component {
```

Comment:
This could be a stateless functional component but of course this is perfectly good as well.

Reply:
Sure. I'll take a stab at that.